defaults/main.yml

```diff
@@ -16,6 +16,10 @@ gitlab_ssl_certificate_key: "/etc/gitlab/ssl/gitlab.key"
 gitlab_create_self_signed_cert: "true"
 gitlab_self_signed_cert_subj: "/C=US/ST=Missouri/L=Saint Louis/O=IT/CN=gitlab"
 
+# Git LFS Configuration.
+gitlab_lfs_enabled: "true"
+gitlab_lfs_storage_path: "/var/opt/gitlab/gitlab-rails/shared/lfs-objects"
```

Comment:
We should _probably_ just have this be `""` in the default vars file, because users could have this anywhere.